tests/__init__.py

```diff
@@ -35,6 +36,33 @@
     "equipment",
 ]
 
+OPTIONS_KEY = "_options"
+
+
+class TestOptions(Enum):
+    CONSISTENT_INGREDIENT_GROUPS = ("consistent_ingredient_groups", True)
+    """
+    Controls if the consistent ingredient groups test is run.
+    Disable if ingredient groups contain sub-quantities of the same ingredient (as the test will fail).
+    """
+
+    def __new__(cls, value: str, default):
+        obj = object.__new__(cls)
+        obj._value_ = value
```

Comment:
@mlduff what's the purpose of this line?

Reply:
@jayaddison I was following this here: https://jwodder.github.io/kbits/posts/multi-value-enum/. I think it is because I am basically overriding the default method, and need to implement the stuff it normally would.

Reply:
Ok, thanks - could you add a comment linking to that from the code, so that future readers understand where it originated from?  I'm not sure I completely understand it yet, so my sense is that that could be the case for other people too.